packages/grafana-ui/src/components/PanelChrome/PanelChrome.tsx

```diff
@@ -124,6 +125,18 @@ export function PanelChrome({
           </h6>
         )}
 
+        {panelNotices && typeof panelNotices.getPanelNotices === 'function' && (
+          <PanelNotices notices={panelNotices.getPanelNotices()} onClick={panelNotices.onClick} />
+        )}
+
+        <PanelDescription description={description} />
+
+        <PanelLinks links={links} />
+
+        <div className={styles.titleItems} data-testid="title-items">
+          {itemsRenderer(titleItems, (item) => item)}
```

Comment:
May be my personal preference, but I don't see this renderer bringing more value than a regular `map` :)

Reply:
This is indeed how the PanelChrome used to render the set of leftItems before we started making changes for the new design. 
Maybe you're right that the value is not exceedingly greater, but `React.Children.toArray()` returns `Array<Exclude<ReactNode, boolean | null | undefined>>` which at least makes sure that we are only rendering valid ReactNodes here.
On the other hand, `titleItems` is being put through `toArray()` and then `filter()` method, whereas otherwise it would just go through one `map()`.